File: FirstMVVM C#/FirstMVVM/Bootstrapper.cs
using Caliburn.Micro;
using FirstMVVM.ViewModels;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows;

namespace FirstMVVM
{
   public class Bootstrapper : BootstrapperBase
    {
        public Bootstrapper()
        {
            Initialize();
        }
        protected override void OnStartup(object sender, StartupEventArgs e)
        {
            DisplayRootViewFor<FirstViewModel>();
        }
    }
}

File: FirstMVVM VB/FirstMVVM/Bootstrapper.vb
Imports Caliburn.Micro

Public Class Bootstrapper
    Inherits BootstrapperBase
    Public Sub New()
        Initialize()
    End Sub
    Protected Overrides Sub OnStartup(sender As Object, e As StartupEventArgs)
        DisplayRootViewFor(Of FirstViewModel)()
    End Sub
End Class

